packages/editor/src/client/routes/comments/commentList.tsx

```diff
@@ -321,57 +200,37 @@ function CommentList() {
             setSortOrder(sortType ?? 'asc')
             setSortField(sortColumn)
           }}>
-          <Column width={350} align="left" resizable>
+          {/* eslint-disable-next-line i18next/no-literal-string */}
+          <Column width={350} align="left" verticalAlign="middle" resizable>
             <HeaderCell>{t('comments.overview.text')}</HeaderCell>
             <Cell dataKey="revisions">
               {(rowData: FullCommentFragment) => (
                 <>
-                  {rowData?.revisions?.length ? (
+                  {rowData.revisions?.length ? (
                     <RichTextBlock
                       displayOnly
                       displayOneLine
                       disabled
                       // TODO: remove this
                       onChange={console.log}
-                      value={rowData?.revisions[rowData?.revisions?.length - 1].text}
+                      value={rowData.revisions[rowData.revisions?.length - 1]?.text || []}
                     />
                   ) : null}
                 </>
               )}
             </Cell>
           </Column>
-          <Column width={150} align="left" resizable>
+          {/* eslint-disable-next-line i18next/no-literal-string */}
```

Comment:
align={'left'} did not work either. Even though, it should according to https://github.com/edvardchen/eslint-plugin-i18next/blob/next/docs/rules/no-literal-string.md